web/src/components/modals/OnboardingModal.tsx

```diff
@@ -66,18 +66,14 @@ export function OnboardingModal() {
   const [hasOnboardingBeenSeen, setHasOnboardingBeenSeen] = useAtom(
     hasOnboardingBeenSeenAtom
   );
-  const [searchParameters] = useSearchParams();
-  const skipOnboarding = searchParameters.get('skip-onboarding') === 'true';
-  const visible = !hasOnboardingBeenSeen && !skipOnboarding;
-
```

Comment:
Does this mean ?skip-onbording won't work anymore? I know some users use this parameter in the wild.

Reply:
Yes it will remove this feature! I'm very curious why people would use this in the wild? 

I think it's good to let the components focus on their functionality and the tests focus on testing as opposed to baking test functionality into components.

Reply:
Not sure exactly why they use it, but they do 😅

Presumably to avoid the onboarding in either saved links (favorites) and/or as a poor man's kiosk mode?

I remember people asking for it when we re-designed the app (but before we had added the functionality back) so I'd suggest keeping the functionality but removing it from the tests, at least until we have a full-fledged kiosk mode.

Reply:
Hmm okay, I'll add it back in

Reply:
A simple kiosk mode indeed! A few companies are showing the app on big screens with no peripherals, so the only way to control this is with query-parameters. On my wish-list is making a dedicated "kiosk mode" that does this and more all at once, but until then we should definitely keep the skip-onboarding param :)